### PR TITLE
travis: update go versions

### PR DESCRIPTION
**Pull request by dvrkps**

### CODE REVIEW

**Comment by ferhatelmas** [comment]
@tleen you didn't ask me but I'd say definitely.

File: .travis.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ sudo: false
 # but since Travis only runs five jobs at a time throughout all of Exercism,
 # it is better citizenship to not test the additional versions.
 go:
-  - 1.7.5
-  - 1.8
+  - 1.8.x
+  - 1.x
```
Comment (ferhatelmas) [inline .travis.yml]: :+1:

File: .travis.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ script:
 matrix:
   include:
     # Keep this in sync with the latest version.
```
Comment (ferhatelmas) [inline .travis.yml]: I believe comment can be removed. 2.x isn't close.

**Comment by tleen** [comment]
> @tleen you didn't ask me but I'd say definitely. 

:flushed: Apologies! I didn't want to spam people with mentions! I'll hit the whole group next time.

**Comment by dvrkps** [comment]
Can I add this in master branch also ?

**Comment by robphoenix** [comment]
> I wonder if we should be merging/rebasing nextercism against master periodically? @robphoenix thoughts?

Yeah, for sure.

**Comment by robphoenix** [comment]
> I've just rebased off master and pushed up to nextercism.

I haven't. I thought I had, but I had just pushed to my own repo `origin` rather than this one `upstream`. 

If one of us rebases `nextercism` on `master`...

```
git fetch upstream master
git rebase upstream/master
```

...then they would have to force push up to this repo. This can have any number of negative consequences, including [attaching that person to many of the commits](https://github.com/exercism/cli/issues/459#issuecomment-329147381) 

Honestly, I'm not sure of the best way to keep the `nextercism` branch in sync with `master`.

I'm going to merge this in to `master`, as the only change requested is not to do with something introduced in this PR and can be dealt with separately. And any conversation concerning branch maintenance can go in a separate issue, rather than spamming this PR.

Thanks again @dvrkps! :tada:

**Pull request merged**